### DISCUSSION TITLE
Should we support multiple versions of openssl? Openssl 1.x and Openssl 3.x or just the latest Openssl 3.x?

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#328] [General]
FYI, OpenSSL 1.1.1 will be going out of service in September: https://www.openssl.org/blog/blog/2023/03/28/1.1.1-EOL/

* We have a draft PR to enable openssl 3.1.1 here: https://github.com/ZOSOpenTools/opensslport/pull/28

**Comment by MikeFultonDev** [comment]
My view would be if there isn’t interest in 1.1 from the community that we move to only support 3.x

**Comment by v1gnesh** [comment]
September is pretty close, just v3 should be fine. 
For once, we can use the reason "because security" appropriately 🙃

**Comment by MikeFultonDev** [comment]
This shift has been done. We are on 3.x now so closing this off.
